src/TiciTacaToeyGameEngine.ts

```diff
@@ -11,12 +11,26 @@ import {
   COMPLETED_GAME_STATUS,
   CalculateWinnerInputType,
   CalculateWinnerOutputType,
+  Player,
+  Timer,
 } from "./model";
 import WebSocket = require("ws");
 import uniq from "lodash.uniq";
 
 const EMPTY_POSITION = "-";
+let timePerPlayer = 5000;
+let firstPlayer;
```

Comment:
This will result in incorrect behavior as multiple games could be updating this variable. Please remove this variable